helm/templates/cloudwatch-agent-daemonset-windows.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if .Values.agent.enabled }}
+{{- $region := .Values.region | required ".Values.region is required." -}}
+apiVersion: cloudwatch.aws.amazon.com/v1alpha1
+kind: AmazonCloudWatchAgent
+metadata:
+  name: {{ template "cloudwatch-agent.name" . }}-windows
+  namespace: {{ .Release.Namespace }}
+spec:
+  podSecurityContext:
+    windowsOptions:
+      hostProcess: true
+      runAsUserName: "NT AUTHORITY\\System"
+  hostNetwork: true
+  image: {{ template "cloudwatch-agent.image" . }}
+  mode: daemonset
+  serviceAccount: {{ template "cloudwatch-agent.serviceAccountName" . }}
+  nodeSelector:
+    kubernetes.io/os: windows
+  config: {{ .Values.agent.windowsDefaultConfig | toJson | quote }}
```

Comment:
customers cant override the default config ?

Reply:
yes, customer won't be able to override default config for Windows since some of applications which are not supported on Windows can crash CW agent on windows altogether. For eg: app signal is not supported on Windows as of now.